.github/workflows/pr-lint-monorepo.yml

```diff
@@ -1,5 +1,7 @@
 name: Run lint checks potentially affecting projects across the monorepo
 on: pull_request
+  branches:
+    - trunk
```

Comment:
I think this [syntax](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request) isn't quite right. I'm thinking its more like this
```
on:
  pull_request:
    branches:    
      - 'trunk'
```

Reply:
Good catch @psealock, I didn't notice that it hadn't been running on this PR. This should be ready for another look.